src/libostree/ostree-repo.c

```diff
@@ -3526,7 +3526,7 @@ ostree_repo_read_commit (OstreeRepo   *self,
 gboolean
 ostree_repo_pull (OstreeRepo               *self,
                   const char               *remote_name,
-                  char                    **refs_to_fetch,
+                  const char              **refs_to_fetch,
```

Comment:
We can't do this with a stable API; people turn these sorts of warnings into errors for good reason.

In the "GLib-and-above" stack, we're not very consistent about this.  The strictly correct thing is more `const char * const*`, which is long and annoying to type.  It's used by e.g. `g_variant_new_strv()`.    But not for example `g_strdupv()`.

Anyways, again we can't change our API, so please rebase without this.